omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -0,0 +1,1732 @@
+            submitValStage.push(validateRequiredField("90209", "5240", "Can Not Be Null", "check_box"));
+
+            if ((getValue("99161" + '.value') == "" &amp;&amp; getValue("99160" + '.value') == "") &amp;&amp; (submitValStageART.indexOf(false) != -1)) {
+                getField("99161" + '.error').html(message7).show();
+                getField("99160" + '.error').html(message8).show();
+            }
+
+            if (submitValStage.indexOf(false) != -1) {
+                submitVal = false;
+                jq().toastmessage('showErrorToast', "  There are errors during the validation of the data on the form. Please correct them and submit again");
+            }
+
+            return submitVal;
+        });
+
+        function enable_disable_ei_final_status(selected_option) {
+
+            var class_name = jq(selected_option).attr("class");
+
+            var length = class_name.length;
+
+            var class_id = parseInt(class_name.substring(length - 1, length));
+
+            var row_id = class_id;
+
+            var disable = true;
+
+            var row_1 = '[class^="ExposedInfantFinalStatus"][class*="Children1"]';
+            var row_2 = '[class^="ExposedInfantFinalStatus"][class*="Children2"]';
+            var row_3 = '[class^="ExposedInfantFinalStatus"][class*="Children3"]';
+            var row_4 = '[class^="ExposedInfantFinalStatus"][class*="Children4"]';
+            var row_5 = '[class^="ExposedInfantFinalStatus"][class*="Children5"]';
+
+            var xx = row_1;
+            var selected_value = jq(selected_option).find(":selected").text();
+
+            if (selected_value.toLowerCase() == "p") {
+                disable = false;
+            }
+            else {
+                disable = true;
+            }
+
+            switch (row_id) {
+                case 1:
+                    xx = row_1;
+                    break;
+                case 2:
+                    xx = row_2;
+                    break;
+                case 3:
+                    xx = row_3;
+                    break;
+                case 4:
+                    xx = row_4;
+                    break;
+                case 5:
+                    xx = row_5;
+                    break;
+            }
+
+
+            jq(xx).each(function () {
+                var group = jq(this);
+                if (class_name.indexOf('Children') == -1) {
+                    /* group.find("input").attr("value", "");*/
+                    group.find("input").attr("disabled", disable);
+
+                    if (disable) {
+                        /* fade out the fields that are disabled*/
+                        group.find("input").fadeTo(250, 0.25);
+                    } else {
+                        /* remove the fade on the fields*/
+                        group.find("input").fadeTo(250, 1);
+                    }
+                }
+
+            });
+
+        }
+
+        function enable_disable_ei_pcr_result(selected_option) {
```

Comment:
@dbaluku Remove this function as this section has changed

Reply:
Done